### PR TITLE
Fixes #36400 - Update ConfigReports to pf4

### PR DESCRIPTION
**Pull request by MariaAga**
The page can be accessed by going to `/config_reports` and clicking one of the values in `Last report`
replace pf3 components with pf4
change class components to arrow components

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/ConfigReports/ConfigReports.js
Comment (kmalyjur) [inline webpack/assets/javascripts/react_app/components/ConfigReports/ConfigReports.js:1]: Everything looks great!

The only thing I have is that in my opinion, the grid might look better if the span was like that:
Metrics: `<GridItem span={4}`
Status: `<GridItem span={6}`
Total: `<GridItem span={2}`
For `Metrics` it's enough space and `Status` overlaps less with `Total` when the screen is smaller.

Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/ConfigReports/ConfigReports.js:1]: Thanks! this does make it look better, fixed

**Comment by adamruzicka** [comment]
@kmalyjur Hi, could you please take a look once more?

**Comment by kmalyjur** [comment]
@adamruzicka everything looks good

**Review by adamruzicka — APPROVED** [review]

**Comment by adamruzicka** [comment]
Thank you @MariaAga & @kmalyjur !

**Pull request merged**